synchronization/build.gradle

```diff
@@ -69,6 +69,7 @@ dependencies {
     testImplementation "org.hamcrest:hamcrest-all:${rootProject.ext.hamcrestVersion}"
 
     testImplementation "org.robolectric:robolectric:${rootProject.ext.robolectricVersion}"
+    androidTestImplementation project(path: ':datacapturing')
```

Comment:
Is this really necessary?

Reply:
I don't see the reason anymore so I reverted this. Thanks